### PR TITLE
Get rand track

### PR DESCRIPTION
**Pull request by AliNough**
Adde my name to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,5 +69,5 @@ Make sure you go through this checklist before submitting your project to Moodle
 ## Authors
 
 - _YOURE_NAME_ (@_your_github_username_)
-- _YOURE_NAME_ (@_your_github_username_)
+- Ali(@_your_github_username_)
```
Comment (S3ak) [inline README.md:72]: Please put your full name

**Review by S3ak — CHANGES_REQUESTED** [review]

**Review by S3ak — CHANGES_REQUESTED** [review]
PLease review my comments and make changes so I can merge this in

File: README.md [hunk 2]
```diff
@@ -67,5 +67,5 @@ Make sure you go through this checklist before submitting your project to Moodle
 ## Authors
 
 - _YOURE_NAME_ (@_your_github_username_)
-- _YOURE_NAME_ (@_your_github_username_)
+- Ali Nough(@AliNough)
```
[the remark this answers is not in the document]
Comment (AliNough) [reply README.md]: Done

File: css/RandStyling.css [hunk 3]
```diff
@@ -0,0 +1 @@
+/* this is an empty css file */
```
[the remark this answers is not in the document]
Comment (AliNough) [reply css/RandStyling.css]: Removed the file and linked randSong.html to original styling file

File: main.js [hunk 4]
```diff
@@ -1,24 +1,45 @@
...
+    "access_token"
+  );
 
 if (accessToken) {
-  // TODO: Hide login button
+  cachedAccessToken = localStorage.setItem("access_token", accessToken);
+
   s.setAccessToken(accessToken);
+  loginContainerEl.classList.add("is-hidden");
+  randNavContainerEl.innerHTML = `
```
[the remark this answers is not in the document]
Comment (AliNough) [reply main.js:24]: i'm confused isn't it already using "is-hidden"?

File: style.css [hunk 5]
```diff
@@ -12,3 +12,4 @@
 @import "css/layout.css";
 @import "css/form.css";
 @import "css/ui.css";
+@import "/css/RandStyling.css";
```
[the remark this answers is not in the document]
Comment (AliNough) [reply style.css]: Removed this import

File: js/randomize.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+function randomizer(yrInput = 0) {
+  // a random number between 0 and 20. replace 20 with response limit
+  resultHolder.innerHTML = "";
+  const randNum = Math.floor(Math.random() * 20);
+  console.log(randNum);
+  //  Problem: the URI is unique for each song, so rand number wont work here.
+  // we need to first get a catalog back and then use the rand number. but how?
+  resultHolder.innerHTML = `
+  <h3>rand.nr: ${randNum}. Year input: ${yrInput}</h3>
```
[the remark this answers is not in the document]
Comment (AliNough) [reply js/randomize.js:37]: Its already commented but it generates a random number between 0 - 20

[the remark this answers is not in the document]
Comment (AliNough) [reply js/randomize.js:37]: I changed it to 0 - 19 for now but later it should be replaced with maximum number of results

**Review by S3ak — APPROVED** [review]

**Pull request merged**